lib/hydra/works/indexers/collection_indexer.rb

```diff
@@ -0,0 +1,9 @@
+module Hydra::Works
+  class CollectionIndexer < Hydra::PCDM::CollectionIndexer
```

Comment:
Since GenericWorks also have child GenericWorks, maybe this should be called ChildGenericWorksIndexer?